python/lsst/daf/butler/core/assembler.py

```diff
@@ -70,11 +76,12 @@ class CompositeAssembler:
         `StorageClass` to be used with this assembler.
     """
 
-    def __init__(self, storageClass):
+    def __init__(self, storageClass: StorageClass):
+        assert storageClass is not None
         self.storageClass = storageClass
 
     @staticmethod
-    def _attrNames(componentName, getter=True):
+    def _attrNames(componentName: str, getter: bool = True) -> Tuple[str, ...]:
         """Return list of suitable attribute names to attempt to use.
```

Comment:
(Pre-existing) inconsistent documentation. I would argue that a list makes more sense than a tuple semantically, since it's an arbitrary-length sequence of like things...